util/reggen/field.py

```diff
@@ -116,6 +125,18 @@ def from_raw(reg_name: str,
         else:
             hwaccess = default_hwaccess
 
+        raw_hwqe = rd.get('hwqe')
+        if raw_hwqe is not None:
```

Comment:
could this just be `if raw_hwqe`?

Reply:
Yes. I admit I simply followed the existing style (same as swaccess and hwaccess).

Now I see we have a style guide, and the linter has lots of complaints about this file :laughing:

Reply:
Ahh, PEP 8 corrects me. It seems this is the correct way. `if raw_hwqe` and `if raw_hwqe is not None` aren't the same--The former may have a bunch of different values that could evaluate to false, but the latter is an explicit comparison to a particular value.